parsec/mca/device/device_gpu.c

```diff
@@ -2126,7 +2126,6 @@ parsec_device_kernel_pop( parsec_device_gpu_module_t   *gpu_device,
         /* We need to manage all data that has been used as input, even if they were read only */
 
         /* Make sure data_in is not NULL */
-        if( NULL == this_task->data[i].data_in ) continue;
```

Comment:
In PTG, we use this to detect that this flow is a control flow. Removing this line will probably create an issue with PTGs with control flows...

Reply:
I removed the line from the PR. I have to check that this doesn't collide with TTG. If it does, I'll open an issue. Should not hold up this PR.